### PR TITLE
fix: Asset profiles UI fixes and updates

### PR DESCRIPTION
**Pull request by arseniy-nikitochkin**
## Summary

This PR includes UI fixes and improvements for asset profile creation and management, along with critical security and accessibility fixes.

#### Asset Profile Management
- Overhauled asset profile creation workflow with improved form handling and state management
- Enhanced issuance draft wizard with better step progression and validation
- Added draft save/restore functionality to preserve user progress
- Improved public info preview with more intuitive UI
- Added confirmation dialog for draft creation with better UX feedback
- Better error handling and form field disabling during save operations

#### UI/UX Improvements
- Refactored classification info rail and asset details configuration
- Enhanced document row rendering and advanced capacities form
- Improved selection cards and form primitives across the issuance flow
- Better visual feedback during form submission
- Improved wizard progress tracking and step navigation

#### Security & Accessibility
- Fixed prototype pollution vulnerability in form handling
- Fixed modal autofocus to ensure proper keyboard accessibility
- Improved form validation to prevent prototype chain pollution

#### Code Quality
- Extracted `use-issuance-draft` hook for better state management
- Added mapping utilities for draft-to-form conversion
- Improved component organization and reusability
- Better TypeScript typing throughout the issuance flow

### Testing
- Recommend testing the complete asset profile creation flow end-to-end
- Verify draft save/restore functionality across sessions
- Test keyboard navigation and modal focus behavior
- Test form validation with edge cases

### Notes
- This is a breaking change for form handling — ensure all dependent code is updated
- Monitor for any prototype pollution edge cases in production
- Consider adding e2e tests for the complete asset profile workflow

### CODE REVIEW

File: apps/sdp-web/src/app/dashboard/issuance/[tokenId]/asset-profile/use-asset-profile-form.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      setShowErrors(false);
+      if (result.assetProfile) {
+        setAssetProfile(result.assetProfile);
+        setBaselineKey(result.assetProfile.updatedAt);
+        setDraft(
+          profileToDraftState(result.assetProfile, {
+            ...token,
+            ...draftTokenPatch(draft),
+          })
+        );
```
[the remark this answers is not in the document]
Comment (arseniy-nikitochkin) [reply apps/sdp-web/src/app/dashboard/issuance/[tokenId]/asset-profile/use-asset-profile-form.ts:112]: Doesn't make sense - after creation we already redirect into asset management. In asset management everything is updated correctly, including token fields

File: apps/sdp-web/src/app/dashboard/issuance/[tokenId]/asset-profile/tabs/overview-tab.tsx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+  onDeploy,
+}: {
+  token: Token;
+  assetProfile: AssetProfile;
+  draft: DraftState;
+  ops: TokenOperations;
+  onDeploy: () => void;
+}) {
+  const category = getCategoryPresentation(assetProfile.assetCategory);
+  const subType = getSubTypePresentation(assetProfile.assetCategory, assetProfile.assetType);
```
[the remark this answers is not in the document]
Comment (arseniy-nikitochkin) [reply apps/sdp-web/src/app/dashboard/issuance/[tokenId]/asset-profile/tabs/overview-tab.tsx:48]: Name is a required field, so I don't see how user can clear it. Also "saved onchain data" for non-deployed token doesn't even make sense. Are you sure?

**Review by GuiBibeau — APPROVED** [review]
